.gitignore

```diff
@@ -83,3 +83,5 @@ silk/tests/test_unit_LPC_inv_pred_gain
 src/Debug
 src/Release
 src/x64
+build
+*.json
```

Comment:
Why?

Remember that cmake uses out-of-src-dir-builds by default, so if you ignore a build folder, all generated json files are inside it and are already ignored. Blanket ignoring all json files may have unintended consequences.

Reply:
Agreed, removed